local/compose/resize.go

```diff
@@ -28,13 +28,13 @@ import (
 	"github.com/docker/docker/pkg/signal"
 )
 
-func (s *composeService) monitorTTySize(ctx context.Context, container string, resize func(context.Context, string, moby.ResizeOptions) error) error {
+func (s *composeService) monitorTTySize(ctx context.Context, container string, resize func(context.Context, string, moby.ResizeOptions) error) {
 	err := resize(ctx, container, moby.ResizeOptions{ // nolint:errcheck
 		Height: uint(goterm.Height()),
 		Width:  uint(goterm.Width()),
 	})
 	if err != nil {
-		return err
```

Comment:
Curious why you dropped the error here?

Reply:
as we monitor resize signal, we don't know when container will terminate and resize would fail.
same as docker/cli@04dad42/cli/command/container/tty.go#L71 which always return nil